developer-guide.md

```diff
@@ -103,21 +103,23 @@ Yes, you can technically request a token for an origin that you don't own. Howev
     Please see [How do I renew a token that is about to expire/has expired?](#renew) for how to renew your tokens 
     to continue participating in the trial.
 
+### 12. Do I need to provide feedback for the trial if I request a token?
```

Comment:
Avoid changing the number/title of existing questions, so add this new entry to the end of the list.

The number and title form part of the link. Where possible, we should preserve the title, so that existing references to specific questions aren't broken.

If a question has an explicit anchor, like the "renew" for question 12, then it's more reasonable to change the title or number.

Reply:
I see - I've updated it to mitigate number changing here.